rules/dropbox_rules/dropbox_linked_team_application_added.py

```diff
@@ -0,0 +1,47 @@
+from panther_base_helpers import deep_get
+
+
+def rule(event):
+    return all(
+        [
+            deep_get(event, "event_type", "_tag", default="Unknown event type tag")
+            == "app_link_team",
+            deep_get(event, "event_type", "description", default="Unknown event type description")
+            == "Linked app for team",
+        ]
+    )
+
+
+def severity(event):
+    # Anything involving non-team members should be High
+    if event.get("involve_non_team_member", ""):
+        return "High"
+    return "Low"
+
+
+def title(event):
+    actor_user_key = [key for key in event.get("actor", {}).keys() if key != "_tag"][0]
+    actor_display_name = deep_get(event, "actor", actor_user_key, "display_name", default="Unknown")
+    return f"Dropbox Team Member Linked App by [{actor_display_name}]"
+
+
+def alert_context(event):
+    additional_user_details = {
+        user_key: user_info
+        for actor_key, actor_value in event.get("actor", {}).items()
+        if actor_key != "_tag"
+        for user_key, user_info in actor_value.items()
+        if user_key not in ("_tag", "display_name")
+    }
```

Comment:
```suggestion
    additional_user_details = {}
    for actor_key, actor_value in event.get("actor", {}).items():
        if actor_key == "_tag":
            continue
        for user_key, user_info in actor_value.items():
            if user_key in ("_tag", "display_name"):
                continue
            additional_user_details[user_key] = user_info
```

it isn't necessary to accept this change for approval, and I think the readability of these chained dictionary comprehensions could be improved by either reformatting to leverage indention or restructured in some other way.

Reply:
Addressed in [`1297205` (#687)](https://github.com/panther-labs/panther-analysis/pull/687/commits/1297205451fb38f8c92a23625fda514f8c5bb864).

I agree that trying to be clever with list/dict comprehensions makes them fairly unreadable.